src/mysky/index.ts

```diff
@@ -121,18 +133,18 @@ export class MySky {
   }
 
   static async New(client: SkynetClient, skappDomain?: string, customOptions?: CustomConnectorOptions): Promise<MySky> {
-    const opts = { ...defaultConnectorOptions, ...customOptions };
+    const opts = { ...DEFAULT_CONNECTOR_OPTIONS, ...customOptions };
 
     // Enforce singleton.
     if (MySky.instance) {
       return MySky.instance;
     }
 
-    let domain = mySkyDomain;
+    let domain = MYSKY_DOMAIN;
     if (opts.alpha) {
```

Comment:
`nit` what if `opts.alpha` and `opts.dev` are set? shouldn't `dev` take precedence?

Reply:
I think setting both should not be allowed. One sends you to the URL for alpha MySky and the other to dev MySky at a different URL. So if you try passing both it should be an error. https://github.com/SkynetLabs/skynet-js/issues/257